packages/network/src/logic/node/StreamManager.ts

```diff
@@ -16,16 +17,16 @@ function keyForDetector({ publisherId, msgChainId }: MessageLayer.MessageID) {
 }
 
 export class StreamManager {
-    private readonly streams: Map<string, StreamState> = new Map<string, StreamState>() // streamKey => {}
+    private readonly streams: Map<SPIDKey,StreamState> = new Map<SPIDKey,StreamState>() // spidKey => {}
```

Comment:
Comment no longer necessary (remnant from pre-TS days, I belive)
```suggestion
    private readonly streams: Map<SPIDKey,StreamState> = new Map<SPIDKey,StreamState>()
```